konig-data-catalog/src/main/java/io/konig/datacatalog/PropertyInfo.java

```diff
@@ -35,7 +37,7 @@
 	private String typeName;
 	private String typeHref;
 	private String description;
-	
+	private List<URI> quantifiedSecurityClassificationList;
```

Comment:
Change the return type to `List<io.konig.datacatalog.Link>` so that we can provide a link from the Security Classification name to the page that describes the given Security Classification.